packages/lumx-react/src/components/select/Select.tsx

```diff
@@ -1,16 +1,23 @@
-import React, { ReactNode, RefObject, useEffect, useRef, useState } from 'react';
+import React, { ReactNode, RefObject, SyntheticEvent, useCallback, useEffect, useRef, useState } from 'react';
 
 import classNames from 'classnames';
 
 import { mdiAlertCircle, mdiCheckCircle, mdiClose, mdiCloseCircle, mdiMenuDown } from '@lumx/icons';
 
-import { Chip, ChipGroup, Dropdown, Emphasis, Icon, IconButton, Placement, Size, Theme } from '@lumx/react';
+import { Emphasis, Size, Theme } from '@lumx/react/components';
+import { IconButton } from '@lumx/react/components/button/IconButton';
+import { Chip } from '@lumx/react/components/chip/Chip';
+import { ChipGroup } from '@lumx/react/components/chip/ChipGroup';
+import { Dropdown } from '@lumx/react/components/dropdown/Dropdown';
+import { Icon } from '@lumx/react/components/icon/Icon';
+import { Placement } from '@lumx/react/components/popover/Popover';
```

Comment:
Is there any reason to use this import style instead of the previous one?

Reply:
For an unknown reason my tests were completely broken with the previous one